website/settings/defaults.py

```diff
@@ -227,7 +227,9 @@ def parent_dir(path):
 ALL_MY_REGISTRATIONS_ID = '-amr'
 ALL_MY_PROJECTS_NAME = 'All my projects'
 ALL_MY_REGISTRATIONS_NAME = 'All my registrations'
-NOTEWORTHY_LINKS_NODE = None
+POPULAR_LINKS_NODE = '8w6qe'  # Temporary.  Needs to be set to public node in production.
+NEW_AND_NOTEWORTHY_LINKS_NODE = 'su3q5' # Temporary. Needs to be set to public node in production.
+QA_USER_IDS = ['nxygz', 'x952z', 'tgak8', 'gaexu', 'rgc49', 'nsx26', 'j52af', 'rbk3c', 'xyubm', 'bje5z', 'twkqb', 'mvzr6', 'dihba'] # Should be updated periodically
```

Comment:
Okay. So I can have QA_USER_IDS = None, with a note instructing an override in local.py?